src/core/net/udp6.cpp

```diff
@@ -87,6 +87,11 @@ Error Udp::Socket::Open(otUdpReceive aHandler, void *aContext)
     return Get<Udp>().Open(*this, aHandler, aContext);
 }
 
+bool Udp::Socket::IsOpened(void) const
```

Comment:
nit: `IsOpen()` seems more correct from an English language perspective.
```suggestion
bool Udp::Socket::IsOpen(void) const
```